experiments/react-native-uifabric/package.json

```diff
@@ -30,21 +30,32 @@
     "@uifabricshared/immutable-merge": "0.2.2",
     "@uifabricshared/theming-ramp": "0.5.2",
     "@uifabricshared/theming-react-native": "0.3.2",
-    "tslib": "^1.9.3"
+    "tslib": "^1.10.0"
```

Comment:
Any reason in particular for this bump? I'm guessing it's fine and there shouldn't be any fallout, but just want to make sure it's intended

Reply:
I took all of the dependencies yarn brought in for oufr (see yarn.lock) as direct dependencies, including this bump.  I also had to add dev dependencies for oufr's devdependencies, which was a long, long list of packages we didn't need for the few we did need to build (e.g. yargs).